frontend/projects/ui/src/app/pages/marketplace-routes/marketplace-list/marketplace-list.page.ts

```diff
@@ -31,7 +32,7 @@ export class MarketplaceListPage {
     .pipe(
       filter(data => exists(data) && !isEmptyObject(data)),
       first(),
-      switchMapTo(this.marketplaceService.getPackages()),
+      switchMap(() => this.marketplaceService.getPackages()),
```

Comment:
This could be the reason why now it requests packages upon reconnection — as opposed to `switchMapTo` now `getPackages` is getting called everytime stream gets here.